src/webrtc/call.ts

```diff
@@ -1379,7 +1433,19 @@ export class MatrixCall extends TypedEventEmitter<CallEvent, CallEventHandlerMap
                             `streamPurpose="${callFeed.purpose}"` +
                             `) to peer connection`,
                     );
+
+                    // this is what would allow us to use addTransceiver(), but it's not available
+                    // on Firefox yet. We call it anyway if we have it.
+                    if (transceiver.sender.setStreams) transceiver.sender.setStreams(this.localUsermediaStream!);
+
                     await oldSender.replaceTrack(track);
+
+                    const parameters = transceiver.sender.getParameters();
+                    transceiver.sender.setParameters({
+                        ...transceiver.sender.getParameters(),
+                        encodings: encodings ?? parameters.encodings,
+                    });
+
```

Comment:
Feels like we're duplicating quite a lot of stuff between here & pushLocalFeed - would be good to re-use some of that, but that doesn't have to be now.

Reply:
I've created https://github.com/vector-im/element-call/issues/877